formatters/logstash/logstash_formatter.go

```diff
@@ -0,0 +1,48 @@
+package logrus_logstash
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/Sirupsen/logrus"
+	"time"
+)
+
+// Formatter generates json in logstash format.
+// Logstash site: http://logstash.net/
+type LogstashFormatter struct {
+	Type string // if not empty use for logstash type field.
+}
+
+func (f *LogstashFormatter) Format(entry *logrus.Entry) ([]byte, error) {
+	entry.Data["@version"] = 1
+	entry.Data["@timestamp"] = entry.Time.Format(time.RFC3339)
```

Comment:
I think the right way is to set `@version` and `@timestamp` fields to user defined values. But in such case maybe problem if user sets incorrect type of fields.
I prefer to leave as is because it's logstash predefined fields.